mock-data/.env

```diff
@@ -7,5 +7,4 @@ PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 # Rpc url for local blockchain node
 RPC_URL=http://127.0.0.1:8545
 
-
```

Comment:
can we remove mock-data folder entirely. we might need it anymore

Reply:
Have removed :)